### ISSUE TITLE
WARNING: Couldn't flush user prefs: java.util.prefs.BackingStoreException: Couldn't get file lock.

### ISSUE THREAD

**Issue by ongjunfeng17** [nus-cs2030/2223-s2#464]
hii, when i run jshell, im getting
WARNING: Couldn't flush user prefs: java.util.prefs.BackingStoreException: Couldn't get file lock.
multiple times

Google tells me "This issue can happen due to not enough hard disk space in the DevTest server or it can be a permissions issue." This led me to check my docker and I realised that the VM Disk storage still has a lot of space available (according to the photo below)

![photo_2023-03-19 17 22 04](https://user-images.githubusercontent.com/120620166/226165739-c09350f9-8295-4342-bdb2-8584d0221741.jpeg)

I was wondering if anyone here has encountered this issue and if so, what exactly is the source of this issue and how may I circumvent it? TIA

P.S. I'm using Macbk Pro M2 Chip and my storage has 378.06 GB available of 494.38 GB

**Comment by nigel-tannn** [comment]
Try restarting your docker.  It'll get rid of the popup messages temporarily but they may come back after a while

**Comment by ongjunfeng17** [comment]
> Try restarting your docker. It'll get rid of the popup messages temporarily but they may come back after a while

Yes I tried restarting my docker before but the popup messages still seem to persist. Was wondering if there was a way to permanently get rid of the popup messages.

**Comment by littlemidget3** [comment]
idt theres a permanent way. Mine comes by once in awhile also, just restart as and when

**Comment by Yew-Wei** [comment]
Usually when this happens I restart my computer and it usually works after. Abit troublesome, looking for other ways to solve this issue as well!

**Comment by ongjunfeng17** [comment]
> Usually when this happens I restart my computer and it usually works after. Abit troublesome, looking for other ways to solve this issue as well!

may i ask after restarting, how long after will the warnings resurface?

**Comment by cocopear** [comment]
from what I can tell, the warnings resurface usually after one or two months from every restart.
